internal/cmd/iam/command_rbac_role_binding.go

```diff
@@ -397,7 +395,11 @@ func (c *roleBindingCommand) displayCCloudCreateAndDeleteOutput(cmd *cobra.Comma
 		if resource != "" {
 			fields = ccloudResourcePatternListFields
 		} else {
-			out.Email = user.GetEmail()
+			if user, err := c.V2Client.GetIamUserById(userResourceId); err != nil {
+				return err
+			} else {
+				out.Email = user.GetEmail()
+			}
```

Comment:
The first branch always exits, no `else` needed.
```suggestion
			user, err := c.V2Client.GetIamUserById(userResourceId)
			if err != nil {
				return err
			}
			out.Email = user.GetEmail()
```